src/test/python/entries.py

```diff
@@ -0,0 +1,134 @@
+import random
+from datetime import datetime, timedelta
+
+import mysql.connector
+import requests
+
+mydb = mysql.connector.connect(
+    host="localhost",
+    user="root",
+    password="password",
+    database="wemove"
+)
+
+myCursor = mydb.cursor()
+
+
+def generate_consistent_entries(userid, start_date, subDuration):
+    date = start_date
+    end_date = start_date + timedelta(days=subDuration)
+
+    if subDuration == 0:
+        randomHour = random.randint(8, 20)
+        entry_time = datetime(date.year, date.month, date.day, randomHour, 0)
```

Comment:
member can arrive at any time, so I think we can add minutes and not set them to 0.
it is a more realistic scenario